### PR TITLE
Add instructions on how to set up env with conda? #276

### LINKED ISSUE #276: Add instructions on how to set up env with conda?
Thank you @Ayushpanditmoto , could you please also add instructions on how to set up env with conda?

_Originally posted by @thinkall in https://github.com/microsoft/autogen/pull/249#discussion_r1360012778_

### PR DESCRIPTION
**Pull request by SDcodehub**
## Why are these changes needed?

Conda env and installation institutions are added

## Related issue number

"Closes #276 "

## Checks

- [x ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ x] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/docs/Installation.md [hunk 1]
```diff
@@ -4,23 +4,31 @@
...
 pip install pyautogen
 ```
-<!--
+
 or conda:
-```
-conda install pyautogen -c conda-forge
-``` -->
+```bash
+conda install -c conda-forge pyautogui
```
Comment (gagb) [inline website/docs/Installation.md:30]: this looks like an error

Comment (SDcodehub) [reply website/docs/Installation.md:30]: tried in conda , works
image
image

**Comment by cieslakd411** [comment]
Should the module be named `pyautogen` instead of `pyautogui` ?

File: website/docs/Installation.md (hunk 1 above)
Comment (qingyun-wu) [reply website/docs/Installation.md:30]: `pyautogui` is a different package from pypi: https://pypi.org/project/PyAutoGUI/

File: website/docs/Installation.md [hunk 2]
```diff
@@ -32,12 +40,19 @@ When running AutoGen out of a docker container, to use docker for code execution
 pip install docker
 ```
 
-* blendsearch
+Or using conda
```
Comment (qingyun-wu) [inline website/docs/Installation.md:43]: can you put the conda options together at the end?

File: website/docs/Installation.md [hunk 3]
```diff
@@ -32,12 +40,19 @@ When running AutoGen out of a docker container, to use docker for code execution
...
 ```
 
-* retrievechat
+* blendsearch, retrievechat
 ```bash
+pip install "pyautogen[blendsearch]"
 pip install "pyautogen[retrievechat]"
 ```
+
+Or using conda
```
Comment (qingyun-wu) [inline website/docs/Installation.md:54]: move the docker conda installation option here as well.

**Comment by qingyun-wu** [comment]
Closing this PR as issue #276 is closed by the other PR. Thanks @SDcodehub, for the contribution!

**Pull request closed without merging**